docs/source/kale.predict.rst

```diff
@@ -30,6 +30,14 @@ kale.predict.losses module
    :undoc-members:
    :show-inheritance:
 
+kale.predict.mlp\_classifier module
```

Comment:
What is the difference with predice.decode --> MLP decoder? Always know existing APIs before adding new ones. Otherwise, lots of duplications and PyKale will become very confusing.

Reply:
merged mlp_classier into mlp_decoder